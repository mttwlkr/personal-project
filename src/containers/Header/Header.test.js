import React from 'react';
import { shallow } from 'enzyme';

describe('Header', () => {

  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Header />)
  })

  it.skip('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  })

  

})