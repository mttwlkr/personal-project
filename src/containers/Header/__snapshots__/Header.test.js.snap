// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Header />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className="nav-btn-div"
>
          <img
                    alt="slvsh-logo"
                    className="slvsh-logo"
                    src="http://d1tajj9tvz4hqc.cloudfront.net/assets/logo_blue_md-94f051d3d0084c821e18d0376a98da8f.png"
          />
          <NavLink
                    activeClassName="active"
                    activeStyle={
                              Object {
                                        "color": "#ff0049",
                                      }
                    }
                    ariaCurrent="true"
                    className="nav-btn nav-games"
                    style={
                              Object {
                                        "textDecoration": "none",
                                      }
                    }
                    to="/games"
          >
                    GAMES
          </NavLink>
          <NavLink
                    activeClassName="active"
                    activeStyle={
                              Object {
                                        "color": "#ff0049",
                                      }
                    }
                    ariaCurrent="true"
                    className="nav-btn nav-riders"
                    style={
                              Object {
                                        "textDecoration": "none",
                                      }
                    }
                    to="/riders"
          >
                    RIDERS
          </NavLink>
          <NavLink
                    activeClassName="active"
                    activeStyle={
                              Object {
                                        "color": "#ff0049",
                                      }
                    }
                    ariaCurrent="true"
                    className="nav-btn nav-create"
                    style={
                              Object {
                                        "textDecoration": "none",
                                      }
                    }
                    to="/create"
          >
                    CREATE
          </NavLink>
</div>,
        <div
          className="login-div"
>
          <NavLink
                    activeClassName="active"
                    activeStyle={
                              Object {
                                        "color": "#ff0049",
                                      }
                    }
                    ariaCurrent="true"
                    className="nav-btn nav-create"
                    style={
                              Object {
                                        "textDecoration": "none",
                                      }
                    }
                    to="/login"
          >
                    LOGIN
          </NavLink>
</div>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <img
              alt="slvsh-logo"
              className="slvsh-logo"
              src="http://d1tajj9tvz4hqc.cloudfront.net/assets/logo_blue_md-94f051d3d0084c821e18d0376a98da8f.png"
/>,
            <NavLink
              activeClassName="active"
              activeStyle={
                            Object {
                                          "color": "#ff0049",
                                        }
              }
              ariaCurrent="true"
              className="nav-btn nav-games"
              style={
                            Object {
                                          "textDecoration": "none",
                                        }
              }
              to="/games"
>
              GAMES
</NavLink>,
            <NavLink
              activeClassName="active"
              activeStyle={
                            Object {
                                          "color": "#ff0049",
                                        }
              }
              ariaCurrent="true"
              className="nav-btn nav-riders"
              style={
                            Object {
                                          "textDecoration": "none",
                                        }
              }
              to="/riders"
>
              RIDERS
</NavLink>,
            <NavLink
              activeClassName="active"
              activeStyle={
                            Object {
                                          "color": "#ff0049",
                                        }
              }
              ariaCurrent="true"
              className="nav-btn nav-create"
              style={
                            Object {
                                          "textDecoration": "none",
                                        }
              }
              to="/create"
>
              CREATE
</NavLink>,
          ],
          "className": "nav-btn-div",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "alt": "slvsh-logo",
              "className": "slvsh-logo",
              "src": "http://d1tajj9tvz4hqc.cloudfront.net/assets/logo_blue_md-94f051d3d0084c821e18d0376a98da8f.png",
            },
            "ref": null,
            "rendered": null,
            "type": "img",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "activeClassName": "active",
              "activeStyle": Object {
                "color": "#ff0049",
              },
              "ariaCurrent": "true",
              "children": "GAMES",
              "className": "nav-btn nav-games",
              "style": Object {
                "textDecoration": "none",
              },
              "to": "/games",
            },
            "ref": null,
            "rendered": "GAMES",
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "activeClassName": "active",
              "activeStyle": Object {
                "color": "#ff0049",
              },
              "ariaCurrent": "true",
              "children": "RIDERS",
              "className": "nav-btn nav-riders",
              "style": Object {
                "textDecoration": "none",
              },
              "to": "/riders",
            },
            "ref": null,
            "rendered": "RIDERS",
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "activeClassName": "active",
              "activeStyle": Object {
                "color": "#ff0049",
              },
              "ariaCurrent": "true",
              "children": "CREATE",
              "className": "nav-btn nav-create",
              "style": Object {
                "textDecoration": "none",
              },
              "to": "/create",
            },
            "ref": null,
            "rendered": "CREATE",
            "type": [Function],
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <NavLink
            activeClassName="active"
            activeStyle={
                        Object {
                                    "color": "#ff0049",
                                  }
            }
            ariaCurrent="true"
            className="nav-btn nav-create"
            style={
                        Object {
                                    "textDecoration": "none",
                                  }
            }
            to="/login"
>
            LOGIN
</NavLink>,
          "className": "login-div",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "activeClassName": "active",
            "activeStyle": Object {
              "color": "#ff0049",
            },
            "ariaCurrent": "true",
            "children": "LOGIN",
            "className": "nav-btn nav-create",
            "style": Object {
              "textDecoration": "none",
            },
            "to": "/login",
          },
          "ref": null,
          "rendered": "LOGIN",
          "type": [Function],
        },
        "type": "div",
      },
    ],
    "type": "header",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="nav-btn-div"
>
            <img
                        alt="slvsh-logo"
                        className="slvsh-logo"
                        src="http://d1tajj9tvz4hqc.cloudfront.net/assets/logo_blue_md-94f051d3d0084c821e18d0376a98da8f.png"
            />
            <NavLink
                        activeClassName="active"
                        activeStyle={
                                    Object {
                                                "color": "#ff0049",
                                              }
                        }
                        ariaCurrent="true"
                        className="nav-btn nav-games"
                        style={
                                    Object {
                                                "textDecoration": "none",
                                              }
                        }
                        to="/games"
            >
                        GAMES
            </NavLink>
            <NavLink
                        activeClassName="active"
                        activeStyle={
                                    Object {
                                                "color": "#ff0049",
                                              }
                        }
                        ariaCurrent="true"
                        className="nav-btn nav-riders"
                        style={
                                    Object {
                                                "textDecoration": "none",
                                              }
                        }
                        to="/riders"
            >
                        RIDERS
            </NavLink>
            <NavLink
                        activeClassName="active"
                        activeStyle={
                                    Object {
                                                "color": "#ff0049",
                                              }
                        }
                        ariaCurrent="true"
                        className="nav-btn nav-create"
                        style={
                                    Object {
                                                "textDecoration": "none",
                                              }
                        }
                        to="/create"
            >
                        CREATE
            </NavLink>
</div>,
          <div
            className="login-div"
>
            <NavLink
                        activeClassName="active"
                        activeStyle={
                                    Object {
                                                "color": "#ff0049",
                                              }
                        }
                        ariaCurrent="true"
                        className="nav-btn nav-create"
                        style={
                                    Object {
                                                "textDecoration": "none",
                                              }
                        }
                        to="/login"
            >
                        LOGIN
            </NavLink>
</div>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <img
                alt="slvsh-logo"
                className="slvsh-logo"
                src="http://d1tajj9tvz4hqc.cloudfront.net/assets/logo_blue_md-94f051d3d0084c821e18d0376a98da8f.png"
/>,
              <NavLink
                activeClassName="active"
                activeStyle={
                                Object {
                                                "color": "#ff0049",
                                              }
                }
                ariaCurrent="true"
                className="nav-btn nav-games"
                style={
                                Object {
                                                "textDecoration": "none",
                                              }
                }
                to="/games"
>
                GAMES
</NavLink>,
              <NavLink
                activeClassName="active"
                activeStyle={
                                Object {
                                                "color": "#ff0049",
                                              }
                }
                ariaCurrent="true"
                className="nav-btn nav-riders"
                style={
                                Object {
                                                "textDecoration": "none",
                                              }
                }
                to="/riders"
>
                RIDERS
</NavLink>,
              <NavLink
                activeClassName="active"
                activeStyle={
                                Object {
                                                "color": "#ff0049",
                                              }
                }
                ariaCurrent="true"
                className="nav-btn nav-create"
                style={
                                Object {
                                                "textDecoration": "none",
                                              }
                }
                to="/create"
>
                CREATE
</NavLink>,
            ],
            "className": "nav-btn-div",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "alt": "slvsh-logo",
                "className": "slvsh-logo",
                "src": "http://d1tajj9tvz4hqc.cloudfront.net/assets/logo_blue_md-94f051d3d0084c821e18d0376a98da8f.png",
              },
              "ref": null,
              "rendered": null,
              "type": "img",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "activeClassName": "active",
                "activeStyle": Object {
                  "color": "#ff0049",
                },
                "ariaCurrent": "true",
                "children": "GAMES",
                "className": "nav-btn nav-games",
                "style": Object {
                  "textDecoration": "none",
                },
                "to": "/games",
              },
              "ref": null,
              "rendered": "GAMES",
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "activeClassName": "active",
                "activeStyle": Object {
                  "color": "#ff0049",
                },
                "ariaCurrent": "true",
                "children": "RIDERS",
                "className": "nav-btn nav-riders",
                "style": Object {
                  "textDecoration": "none",
                },
                "to": "/riders",
              },
              "ref": null,
              "rendered": "RIDERS",
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "activeClassName": "active",
                "activeStyle": Object {
                  "color": "#ff0049",
                },
                "ariaCurrent": "true",
                "children": "CREATE",
                "className": "nav-btn nav-create",
                "style": Object {
                  "textDecoration": "none",
                },
                "to": "/create",
              },
              "ref": null,
              "rendered": "CREATE",
              "type": [Function],
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <NavLink
              activeClassName="active"
              activeStyle={
                            Object {
                                          "color": "#ff0049",
                                        }
              }
              ariaCurrent="true"
              className="nav-btn nav-create"
              style={
                            Object {
                                          "textDecoration": "none",
                                        }
              }
              to="/login"
>
              LOGIN
</NavLink>,
            "className": "login-div",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "activeClassName": "active",
              "activeStyle": Object {
                "color": "#ff0049",
              },
              "ariaCurrent": "true",
              "children": "LOGIN",
              "className": "nav-btn nav-create",
              "style": Object {
                "textDecoration": "none",
              },
              "to": "/login",
            },
            "ref": null,
            "rendered": "LOGIN",
            "type": [Function],
          },
          "type": "div",
        },
      ],
      "type": "header",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
